scripts/clonebuild.sh

```diff
@@ -29,39 +24,25 @@ DEPLOY_GIT_BRANCH=${@:$OPTIND+2:1}
 DEEP_CLEAN=${@:$OPTIND+3:1}
 NAMED_BRANCH=${@:$OPTIND+4:1}
 
+# Remove the clone folder
+if [ $DEEP_CLEAN = true ]; then
+  make cleanall
+fi
```

Comment:
we want to `cleanall` in the snapshot not locally.

Reply:
`cleanall` remove the entire snapshot

Reply:
Ok I hadn t see that. But still you are in the current directory and your also removing your built artefacts. Just remove the clonedir instead.

Reply:
yes good idea

Reply:
done